frontend/package.json

```diff
@@ -25,6 +25,7 @@
     "@types/react-dom": "18.2.15",
     "autoprefixer": "10.4.16",
     "axios": "^1.5.1",
+    "chart.js": "^4.4.0",
```

Comment:
where are we using charts?

Reply:
For Creating piecharts in the file [CustomPiechart.tsx]